server/chat.ts

```diff
@@ -1115,6 +1115,11 @@ export class CommandContext extends MessageContext {
 					targetUser.id !== user.id) {
 					throw new Chat.ErrorMessage(this.tr`You are blocking private messages right now.`);
 				}
+				if (user.settings.blockChallenges && (user.settings.blockChallenges === true ||
+					!Users.globalAuth.atLeast(targetUser, user.settings.blockChallenges)) && !targetUser.can('bypassblocks') &&
+					targetUser.id !== user.id) {
+					throw new Chat.ErrorMessage(this.tr`You are blocking challenge right now.`);
+				}
```

Comment:
```suggestion
				if (
					user.settings.blockChallenges && 
					(user.settings.blockChallenges === true || !Users.globalAuth.atLeast(targetUser, user.settings.blockChallenges)) &&
					!targetUser.can('bypassblocks') &&
					targetUser.id !== user.id
				) {
					throw new Chat.ErrorMessage(this.tr`You are blocking challenges right now.`);
				}
```

Reply:
Note that "You are blocking challenges right now." will need to be updated in all the translations files.